parsoid/Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /home/parsoid
 
 RUN git clone https://gerrit.wikimedia.org/r/p/mediawiki/services/parsoid && \
     cd parsoid && \
-    git checkout v0.8.1 && \
+    git checkout v0.10.0 && \
```

Comment:
Took the opportunity to bump node and parsoid, but this is unrelated and shouldn't make a difference.